README.md

```diff
@@ -1,3 +1,295 @@
+* You need to have permissions to access your base cluster's private key
+* You are connected to your Kubernetes cluster (environment variable `KUBECONFIG` is set)
+  * [Viewing kubeconfig (GKE)](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl#viewing_kubeconfig)
+  * [Create a kubeconfig for Amazon EKS](https://docs.aws.amazon.com/eks/latest/userguide/create-kubeconfig.html)
+  * [Use Azure role-based access controls to define access to the Kubernetes configuration file in Azure Kubernetes Service (AKS)](https://docs.microsoft.com/en-us/azure/aks/control-kubeconfig-access)
+
+## Procedure
+
+To install Gardener in your base cluster, a command line tool [sow](https://github.com/gardener/sow) is used. It depends on other tools to be installed. To make it simple, we have created a Docker image that already contains `sow` and all required tools. To execute `sow` you call a [wrapper script](https://github.com/gardener/sow/tree/master/docker/bin) which starts `sow` in a Docker container (Docker will download the image from [eu.gcr.io/gardener-project/sow](http://eu.gcr.io/gardener-project/sow) if it is not available locally yet). Docker executes the sow command with the given arguments, and mounts parts of your file system into that container so that `sow` can read configuration files for the installation of Gardener components, and can persist the state of your installation. After `sow`'s execution Docker removes the container again.
+
+1. Clone the `sow` repository and add the path to our [wrapper script](https://github.com/gardener/sow/tree/master/docker/bin) to your `PATH` variable so you can call `sow` on the command line.
+
+    ```bash
+    # setup for calling sow via the wrapper
+    git clone "https://github.com/gardener/sow"
+    cd sow
+    export PATH=$PATH:$PWD/docker/bin
+    ```
+
+1. Create a directory `landscape` for your Gardener landscape and clone this repository into a subdirectory called `crop`:
+
+    ```bash
+    cd ..
+    mkdir landscape
+    cd landscape
+    git clone "https://github.com/gardener/garden-setup" crop
+    ```
+
+1. If you created the base cluster using GKE you need to convert your `KUBECONFIG` to one that uses basic authentication with Google-specific configuration parameters:
+
+    1. Set your `KUBECONFIG` environment variable to a file that does not exist.
+
+    1. Download the GKE kubeconfig:
+
+        ```bash
+        gcloud container clusters get-credentials <your_cluster> --zone <your_zone> --project <your_project>
+        ```
+
+    1. Save the GKE kubeconfig in a local file (for example `kubekonfig`) in your `landscape` folder. Make sure that you assign this file to field `landscape.cluster.kubeconfig` in the configuration file `acre.yaml` that is created in the next step.
+    > The filename and path you will use in file `acre.yaml` should be *different* from the file you have set in your `KUBECONFIG` environment variable.
```

Comment:
@Diaphteiros: Wouldn't it make more sense for the user then to set the KUBECONFIG variable to the actual `kubeconfig` file that is overwritten to access the cluster? Or does the variable KUBECONFIG still have to point to a file that does not exist before you execute `sow covertkubeconfig`?

If KUBECONFIG may reference the created file I would get rid of sub-step 1, and instead of the comment I'd add a step that asks users to set the KUBECONFIG variable to the created file.

Thanks

Reply:
The `KUBECONFIG` variable isn't evaluated at all in `sow convertkubeconfig`. It doesn't matter where it points to. 
Actually, since the `sow` command is run inside a docker container, it has a completely different `KUBECONFIG` anyway.
So this can be simplified to pointing the `KUBECONFIG` to the (non-existing) `landscape/kubeconfig`, then the `gcloud` command and then `sow convertkubeconfig` (GKE scenario).

Reply:
@Diaphteiros : Ok, understood.

Nevertheless, all users need to set their `KUBECONFIG` environment variable to be able to do the `watch kubectl` later. 

They might have done that in some shell-rc file, but to be sure no one stumbles over this I added it as an explicit step (step 7), and added a recommendation to save it in your landscape directory (step 4). 

That way, we solve the "hen and egg" problem and the only GKE specific step that remains really is step 6. 

Hope you like my reorganization - otherwise let's have a short call.